pkg/k8s/common.go

```diff
@@ -152,33 +152,31 @@ func newContainer(port int, image string, containerPorts []apiv1.ContainerPort,
 	}
 }
 
-func newDeployment(namespace, name string, port int, image string, ports []apiv1.ContainerPort, selector map[string]string, cert, key string) *appsv1.Deployment {
+func newDeployment(namespace, name string, port int, image string, ports []apiv1.ContainerPort, selector map[string]string, deploymentLabels map[string]string, cert, key string) *appsv1.Deployment {
 	replicas := int32(1)
+	labels := map[string]string{
+		"app.kubernetes.io/name":     name,
+		"app.kubernetes.io/instance": name,
+	}
+	for key, value := range deploymentLabels {
```

Comment:
instead of iterating deploymentLabels, why not add `app.kubernetes.io/name` and `app.kubernetes.io/instance` and pass it to the Deployment object?